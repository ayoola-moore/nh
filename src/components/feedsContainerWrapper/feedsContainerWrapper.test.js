import React from "react";
import FeedContainerWrapper from "./"

describe("ErrorBoundary", () => {
  const component = shallow(<FeedContainerWrapper />);
  it("should render FeedContainerWrapper properly", () => {
    expect(component).toMatchSnapshot();
  });
});
