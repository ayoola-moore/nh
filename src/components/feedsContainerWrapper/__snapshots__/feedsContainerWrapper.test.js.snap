// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary should render FeedContainerWrapper properly 1`] = `ShallowWrapper {}`;
